packages/react-composites/src/composites/CallWithChatComposite/CallWithChatControlBar.tsx

```diff
@@ -166,6 +166,7 @@ export const CallWithChatControlBar = (props: CallWithChatControlBarProps & Cont
                     data-ui-id="call-with-chat-composite-more-button"
                     strings={moreButtonStrings}
                     onClick={props.onMoreButtonClicked}
+                    disabled={props.disableButtonsForLobbyPage}
```

Comment:
I'm concerned there's still an underlying issue that the chat and people pane could be opened when not in a call yet from somewhere else (in the future) that we won't be protecting against.
Might be an over optimization but we could consider when we call `setActivePane(chat | people)` first ensuring that we are actually in a call (checking the `hasJoinedCall` variable) and setting the active pane unless we have joined a call, open to other suggestions or simply not doing this for now!

This fix should definitely should be kept though - nice simply fix too!

Reply:
disableButtonsForLobbyPage={isInLobbyOrConnecting} ,const isInLobbyOrConnecting = currentPage === 'lobby'; this prop should be already detecting if we are in a call?

Reply:
It will, and this will definitely solve the bug today!

My thinking is we still have a hole where a user can hit a totally blank page that might come up in the future. E.g. if we add a `CallWithChatComposite.openPane('people')` function then this could be called at a bad time causing the user to hit a blank white page. Or maybe we add the ability to click on your own tile and it opens the people pane from some option on your tile, we could uncover this issue again accidentally because its hard to spot and very missable in manual testing.
So might be worth putting in extra safeguarding today to avoid a regression that causes this to happen in the future, but also feel free to leave this be - totally up to you!

Reply:
Nice this has been fixed with the latest hasJoinedCall checks! Thanks!